### PR TITLE
SAMPLE APP Upgrade Gradle 8.1.1

### PR DESCRIPTION
**Pull request by RafaRuiz**
Upgrade Gradle 8.1.1

### CODE REVIEW

File: build.gradle [hunk 1]
```diff
@@ -15,7 +15,7 @@ buildscript {
             'guava': '27.1',
             'findbugs': '3.0.2',
             'joda': '2.10.1',
-            'jackson': '2.14.1',
+            'jackson': '2.14.2',
```
Comment (mpbarnwell) [inline build.gradle:18]: If you're going to start bumping dependencies, you should probably do them all (in a separate PR), but if you're going to bump just this one you should probably bump to `2.14.3`

File: gradle/wrapper/gradle-wrapper.properties [hunk 2]
```diff
@@ -1,6 +1,6 @@
-#Wed Apr 24 23:57:46 CEST 2019
+#Fri Sep 15 11:38:13 CEST 2023
 distributionBase=GRADLE_USER_HOME
 distributionPath=wrapper/dists
 zipStoreBase=GRADLE_USER_HOME
 zipStorePath=wrapper/dists
-distributionUrl=https\://services.gradle.org/distributions/gradle-7.5-all.zip
+distributionUrl=https\://services.gradle.org/distributions/gradle-8.0-bin.zip
```
Comment (mpbarnwell) [inline gradle/wrapper/gradle-wrapper.properties:6]: This should match the version listed in `build.gradle`/buildtools, so 8.1.1, or you should just go for the latest version, if it works.

File: ui/build.gradle [hunk 3]
```diff
@@ -73,10 +79,10 @@ dependencies {
     annotationProcessor "org.immutables:value:2.7.5"
     testAnnotationProcessor "org.immutables:value:2.7.5"
 
-    implementation('com.google.guava:guava:27.1-android') { force = true }
+    implementation 'com.google.guava:guava:27.1-android'
```
Comment (mpbarnwell) [inline ui/build.gradle:82]: Guava is listed in the parent build.gradle ext.versions list, so you should reference that? Also this version definitely needs upgrading, again, happy for that to be a separate PR.

File: ui/src/main/java/com/doordeck/sdk/common/manager/Doordeck.kt [hunk 4]
```diff
@@ -24,6 +23,7 @@ import com.doordeck.sdk.ui.qrcode.QRcodeActivity
 import com.doordeck.sdk.ui.unlock.UnlockActivity
 import com.doordeck.sdk.ui.unlock.UnlockActivity.Companion.COMING_FROM_DIRECT_UNLOCK
 import com.doordeck.sdk.ui.verify.VerifyDeviceActivity
+import com.github.doordeck.ui.BuildConfig
```
Comment (mpbarnwell) [inline ui/src/main/java/com/doordeck/sdk/common/manager/Doordeck.kt:26]: Import needed?

Comment (RafaRuiz) [reply ui/src/main/java/com/doordeck/sdk/common/manager/Doordeck.kt:26]: Yup. Namespaces were needed now and the `R` and `BuildConfig` classes changed
image

File: gradle/wrapper/gradle-wrapper.properties (hunk 2 above)
Comment (RafaRuiz) [reply gradle/wrapper/gradle-wrapper.properties:6]: Done

File: ui/build.gradle (hunk 3 above)
Comment (RafaRuiz) [reply ui/build.gradle:82]: Done

File: build.gradle (hunk 1 above)
Comment (RafaRuiz) [reply build.gradle]: Done. The only ones changed relate to migrating to Gradle over 8.x
